colorcoder/Main.java

```diff
@@ -1,120 +1,39 @@
-package colorcoder;
+	package colorcoder;
 
 public class Main {
-    enum MajorColor {
```

Comment:
Class Name and Method Name is same  -  what is the benefit of ambiguity 
Main class holds too many responsibilities